src/frontend-samples/snow-decoration-sample/README.md

```diff
@@ -0,0 +1,28 @@
+# Particle Effect - Snow & Rain
```

Comment:
This readme should probably describe the difference between rain (static) and rain (dynamic)